solutions/aspnet_core/aspnet_samples/Makefile

```diff
@@ -49,22 +49,32 @@ run-todo:
 	./client.sh Todo
 	./kill.sh Todo
 
+run-podcast-with-retry:
+	./run_podcast_with_retry.sh
```

Comment:
You can alternatively use scripts/myst-retry. It provides similar functionality.

Reply:
Thanks for sharing! Will try to use this tool in the future